pkgs/applications/office/espanso/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenv
+, fetchFromGitHub
+, rustPlatform
+, pkgconfig
+, extra-cmake-modules
+, libX11
+, libXi
+, libXtst
+, libnotify
+, openssl
+, xclip
+, xdotool
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "espanso";
+  version = "0.6.3";
+
+  src = fetchFromGitHub {
+    owner = "federico-terzi";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1x5p7hniapggqd18rx26mjvdf33z7rm7zz5vsqm2siv3mcl19033";
+  };
+
+  cargoSha256 = "0liwwdncymjql5dw7rwhhimcr7qdbyvfgmsd0bawvi0ym7m1v408";
+
+  nativeBuildInputs = [
+    extra-cmake-modules
+    pkgconfig
+  ];
+
+  buildInputs = [
+    libX11
+    libXtst
+    libXi
+    libnotify
+    openssl
+    xdotool
+  ];
+
+  # Some tests require networking
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Cross-platform Text Expander written in Rust";
+    homepage = "https://espanso.org";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```
I think this should probably be gpl3Plus, since the source files have the *either version 3 of the License, or (at your option) any later version* wording.

---

Based on similar PR comments, it should yes. I just updated the file accordingly.

---

Awesome, thanks a lot!